nextflow_schema.json

```diff
@@ -96,26 +96,26 @@
             "description": "These parameters influence the database to be used in classifying the taxonomy.",
             "default": "",
             "properties": {
-                "taxa_classification_mmseqs_databases_localpath": {
+                "taxa_classification_mmseqs_db": {
```

Comment:
I would set the `Taxonomic classifcication` general thing to `Taxonomic classificaton: general options` like the BGC one (see below, if you change the variable/entry name, make sure to update that everywhere)